packages/web/index.js

```diff
@@ -165,6 +164,12 @@ module.exports = (neutrino, opts = {}) => {
           Object
             .keys(neutrino.options.mains)
             .forEach(key => {
+              neutrino.use(htmlTemplate, merge({
+                pluginId: `html-${key}`,
+                filename: `${key}.html`,
+                chunks: [key, 'vendor', 'runtime']
```

Comment:
Could you add a comment here and to `packages/chunk/index.js` saying the two sets of chunk names need to be kept in sync?

Reply:
Sure thing!